client/app/widgets/Notifications/index.jsx

```diff
@@ -18,25 +18,26 @@ export type State = {
   signedInKey?: number,
 };
 
-export class Notifications extends React.Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = { notifications: '', alreadyMounted: false, open: false };
-  }
+const Notifications = ({
+  element,
+}: Props) => {
+  const [notifications, setNotifications] = useState('');
+  const [alreadyMounted, setAlreadyMounted] = useState(false);
+  const [open, setOpen] = useState(false);
+  const [modalKey, setModalKey] = useState('');
```

Comment:
Since this prop is not required, I don't think we need to set the initial state to a string. This should be `undefined` instead. I think this may be responsible for some issues you're having with linting and likely the tests.

Reply:
Ok I'll make the required changes. Thank you!